libexec/swiftenv-install

```diff
@@ -193,21 +193,44 @@ build_version() {
 }
 
 find_binary_url_from_api() {
-  vlog "Checking for a URL for the $VERSION on $(get_platform)."
-
-  status_code=$(curl -o /dev/null --silent --head --write-out '%{http_code}' "https://swiftenv-api.fuller.li/versions/$VERSION/binaries/$(get_platform)" || true)
+  vlog "Checking for a URL for Swift $VERSION on $(get_platform)."
+
+  case $(get_platform) in
```

Comment:
I'd suggest perhaps we don't special case each version and generically handle Ubuntu (replacing the `.` out if we need to handle both variables).

This will prevent any maintenance work needed for any future versions of Ubuntu that get released.

Reply:
@adam-fowler Here's what I was thinking:

```
  "ubuntu"* )
    SWIFT_BUILD_FOLDER=${platform/.}
    SWIFT_ZIP_FILE=$platform
```

need to bind `platform=$(get_platform)` above and then use `$platform` for the case (so we have the variable for our use below). Here we'd accept all "versions" and then remove the dot for build folder.

Reply:
Done